scripts/updateRiskOracleConfig.ts

```diff
@@ -1,51 +1,85 @@
 import hre from "hardhat";
-
 import { encodeData } from "../utils/hash";
+import { getFullKey } from "../utils/config";
 import * as keys from "../utils/keys";
 
 export async function main() {
+  const riskOracleConfig = await hre.gmx.getRiskOracle();
+
+  const dataStore = await hre.ethers.getContract("DataStore");
   const config = await hre.ethers.getContract("Config");
 
-  const syncConfigKeys = [
-    // {
-    //   baseKey: keys.SYNC_CONFIG_MARKET_DISABLED,
-    //   data: encodeData(["address"], ["0x352f684ab9e97a6321a13CF03A61316B681D9fD2"]),
-    // },
-    {
+  if (!riskOracleConfig) {
+    throw new Error("No configuration found for the current network");
+  }
+
+  const syncConfigKeys = [];
+  
+  for (const marketAddress in riskOracleConfig.markets) {
+    const marketConfig = riskOracleConfig.markets[marketAddress];
+    syncConfigKeys.push({
```

Comment:
could you check how we handle these in e.g. updateMarketConfig https://github.com/gmx-io/gmx-synthetics/blob/updates/scripts/updateMarketConfigUtils.ts#L507

it first reads all values from DataStore https://github.com/gmx-io/gmx-synthetics/blob/updates/scripts/updateMarketConfigUtils.ts#L528

then it updates params if only it doesn't match values in DataStore https://github.com/gmx-io/gmx-synthetics/blob/updates/scripts/updateMarketConfigUtils.ts#L548

without this current script will always send transaction

Reply:
Ok, but I already included a check for this in updateRiskOracleConfig.ts: 
https://github.com/gmx-io/gmx-synthetics/blob/config-syncer/scripts/updateRiskOracleConfig.ts#L52-L65 

This is similar to how it was done in updateOracleConfig.ts: 
https://github.com/gmx-io/gmx-synthetics/blob/config-syncer/scripts/updateOracleConfig.ts#L19-L23

Maybe I'm missing something, but isn't this already addressed with the highlighted code? Or maybe the issue is that I'm doing this check after retrieving all the markets, market parameters and parameters from riskOracle.ts?